README.md

```diff
@@ -83,6 +83,7 @@
 This section includes software, guides and tools that are not *specifically* designed to work with Jellyfin, but can be useful for tasks related to media management or to enhance Jellyfin's functionality.
 
 - [calibre2jellyfin](https://github.com/shawn61cp/calibre2jellyfin) - Python script to construct a Jellyfin ebook library from a Calibre library.
+- [Cloud Seeder](https://ipv6.rs/cloudseeder) - A 1-click installer and maintainer for Jellyfin on Windows, Mac and Linux.
```

Comment:
Hi, we would prefer to use the GitHub link instead:
```suggestion
- [Cloud Seeder](https://github.com/ipv6rslimited/cloudseeder) - A 1-click installer and maintainer for Jellyfin on Windows, Mac and Linux.
```

Reply:
My apologies! I wasn't aware there was actually a button that allows us to commit a suggestion; so I went ahead and made a pull request. By the time I realized you could click a commit suggestion button, it was already too late. I hope this is alright!

Also, thank you for the suggestion as it makes more sense.